tests/dummy/app/routes/index.js

```diff
@@ -0,0 +1,86 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+	model() {
+		return { events: fixture };
+	}
+});
```

Comment:
lines 4-6 have tabs; need to convert to spaces (I know silly right, tabs and spaces have friends so long)